patterns.html

```diff
@@ -37,22 +38,15 @@ <h3 class="article-section__title">{{ category.pattern-category }}</h3>
             </ul>
           </li>
         </ul>
-        {% for pattern in category.patterns | sort: 'title' %}
```

Comment:
Since the sorting method was causing an error, I deleted the duplicate block and fixed it once